smokescreen.go

```diff
@@ -122,9 +123,12 @@ func buildProxy() *goproxy.ProxyHttpServer {
 			ctx.Resp = errorResponse(ctx.Req, err)
```

Comment:
My reading of the code suggests we might never print this? It would be good to know that we're failing lookups but a metric might also be fine.